WalletWasabi.Gui/App.xaml

```diff
@@ -33,5 +35,52 @@
     <DataTemplate DataType="id:IView">
       <cont:ViewModelViewHost DataContext="{Binding Context}" />
     </DataTemplate>
+
+    <DataTemplate DataType="sys:Exception">
```

Comment:
@danwalmsley Do you know of anyway that we can move these `DataTemplate`s to `Styles.xaml`?